src/lumigo_tracer/parsers/event_parser.py

```diff
@@ -0,0 +1,93 @@
+import os
+import re
+from collections import OrderedDict
+from typing import Dict, List
+
+from lumigo_tracer.parsers.utils import str_to_tuple, str_to_list
+from lumigo_tracer.utils import get_logger
+
+
+API_GW_REGEX = re.compile(rf".*execute-api.*amazonaws\.com.*")
+API_GW_KEYS_ORDER = str_to_list(os.environ.get("LUMIGO_API_GW_KEYS_ORDER", "")) or [
+    "resource",
+    "path",
+    "httpMethod",
+    "queryStringParameters",
+    "multiValueQueryStringParameters",
+    "pathParameters",
+    "body",
+    "requestContext",
+    "headers",
+]
+API_GW_PREFIX_KEYS_HEADERS_DELETE_KEYS = str_to_tuple(
+    os.environ.get("LUMIGO_API_GW_PREFIX_KEYS_HEADERS_DELETE_KEYS", "")
+) or ("cookie", "X-Amz", "Accept", "CloudFront", "Via", "X-Forwarded", "sec-")
+API_GW_REQUEST_CONTEXT_FILTER_KEYS = str_to_list(
+    os.environ.get("LUMIGO_API_GW_REQUEST_CONTEXT_FILTER_KEYS", "")
+) or ["authorizer"]
+API_GW_KEYS_DELETE_KEYS = str_to_list(os.environ.get("LUMIGO_API_GW_KEYS_DELETE_KEYS", "")) or [
+    "multiValueHeaders"
+]
+
+
+class EventParseHandler:
+    def is_supported(self, event) -> bool:
+        raise NotImplementedError()
+
+    def parse(self, event):
+        raise NotImplementedError()
+
+
+class ApiGWHandler(EventParseHandler):
+    def is_supported(self, event) -> bool:
+        if event.get("requestContext") and event.get("requestContext").get("domainName"):
```

Comment:
```suggestion
        if event.get("requestContext") and event.get("requestContext", {}).get("domainName"):
```
So if it doesn't exists, then it will not crush on NoneType...
add test!